package.json

```diff
@@ -42,7 +42,7 @@
     "check-staged": "lint-staged",
     "prettier": "prettier --write '**/*.{js,jsx}'",
     "pretest": "NODE_ENV=test npm run lint && npm run prettier && npm run db:migrate:testdb && npm run seedall",
-    "test": "NODE_ENV=test nyc mocha --recursive src/tests --require @babel/register --exit",
+    "test": "NODE_ENV=test nyc mocha --recursive src/tests --require @babel/register --exit --no-timeouts",
```

Comment:
why did we add `--no-timeouts` option to our test?

Reply:
This may have been a mistake I made while rebasing, but it has been resolved now